Source/Libraries/GSF.Core/Data/DataInserter.cs

```diff
@@ -413,15 +413,12 @@ private bool ClearTable(Table table)
                 useTruncateTable = m_forceTruncateTable || (table.Parent.Parent.DataSourceType == DatabaseType.SQLServer && !table.ReferencedByForeignKeys);
             }
 
-            if (useTruncateTable)
-                deleteSql = "TRUNCATE TABLE " + table.SQLEscapedName;
-            else
-                deleteSql = "DELETE FROM " + table.SQLEscapedName;
-
             try
             {
-                table.Connection.ExecuteNonQuery(deleteSql, Timeout);
-
+                if (useTruncateTable)
+                    table.Connection.ExecuteNonQuery("TRUNCATE TABLE {0}", table.SQLEscapedName, Timeout);
+                else
+                    table.Connection.ExecuteNonQuery("DELETE FROM {0}", table.SQLEscapedName, Timeout);
```

Comment:
Parameter substitution does not work on Tables

Reply:
updated